app/views/escorts/confirm_cancel.html.slim

```diff
@@ -0,0 +1,18 @@
+= content_for :breadcrumbs do
+  - breadcrumbs_for_page root: true do
+    - breadcrumb detainee_breadcrumb(escort.detainee), escort_path(escort)
+
+.escort.per-page
+  h1
+    = t('.cancelling_per_for')
```

Comment:
Isn't this just the title for the cancel escort form? Is there a need to have the locale key to match what the actual translation is? Shouldn't this just be something like `form.cancel_escort.title` or something similar?